patched_bdist_wheel.py

```diff
@@ -0,0 +1,103 @@
+"""
```

Comment:
my reasoning: it's similar to setupext.py, so should be next to it. I could add that there or name it `setup_patches.py`?